components/applications-service/pkg/storage/postgres/hab_event.go

```diff
@@ -201,7 +201,7 @@ func (db *Postgres) insertNewServiceFromHealthCheckEvent(
 		}
 
 		// 4) Service
-		svc := newService(pkgIdent, health, did, sid, gid)
+		svc := newService(pkgIdent, health, did, sid, gid, svcMetadata.GetPackageIdent())
```

Comment:
You already have the package ident here, it is the first argument in this function.

Reply:
That is a struct of it broken apart though. Maybe I can refactor further.

Reply:
Ok, I moved the full string into the package ident struct so that we don't have a new param here. Let me know what you think.